[2021-12-25 11:23:06,210] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: survey.myfunction 2021-12-25T11:23:03.747287+00:00 [queued]>
[2021-12-25 11:23:06,243] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: survey.myfunction 2021-12-25T11:23:03.747287+00:00 [queued]>
[2021-12-25 11:23:06,244] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-12-25 11:23:06,245] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-12-25 11:23:06,245] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-12-25 11:23:06,273] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): myfunction> on 2021-12-25T11:23:03.747287+00:00
[2021-12-25 11:23:06,281] {standard_task_runner.py:52} INFO - Started process 64 to run task
[2021-12-25 11:23:06,290] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'survey', 'myfunction', '2021-12-25T11:23:03.747287+00:00', '--job-id', '187', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/survey.py', '--cfg-path', '/tmp/tmpnxnwgaek', '--error-file', '/tmp/tmp2gciwhay']
[2021-12-25 11:23:06,293] {standard_task_runner.py:77} INFO - Job 187: Subtask myfunction
[2021-12-25 11:23:06,370] {logging_mixin.py:104} INFO - Running <TaskInstance: survey.myfunction 2021-12-25T11:23:03.747287+00:00 [running]> on host e1668ec19f6d
[2021-12-25 11:23:06,513] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=survey
AIRFLOW_CTX_TASK_ID=myfunction
AIRFLOW_CTX_EXECUTION_DATE=2021-12-25T11:23:03.747287+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-12-25T11:23:03.747287+00:00
[2021-12-25 11:23:06,532] {python.py:118} INFO - Done. Returned value was: None
[2021-12-25 11:23:06,587] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=survey, task_id=myfunction, execution_date=20211225T112303, start_date=20211225T112306, end_date=20211225T112306
[2021-12-25 11:23:06,712] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-12-25 11:23:06,747] {local_task_job.py:146} INFO - Task exited with return code 0
